llvm/tools/sycl-post-link/SYCLDeviceLibReqMask.h

```diff
@@ -17,8 +17,12 @@
 #pragma once
 
 #include "llvm/Pass.h"
+
+#include <cstdint>
+#include <string>
 #include <unordered_map>
```

Comment:
These headers may be moved to cpp as well.

Reply:
Thank you, I've moved the headers to .cpp.